applications/asset_tracker_v2/doc/cloud_wrapper.rst

```diff
@@ -0,0 +1,162 @@
+.. _api_cloud_wrapper:
+
+Cloud wrapper API
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The cloud wrapper API is a generic API used to control the connection to a supported |NCS| client library through :ref:`Integration layers <integration_layers>`.
+It exposes generic functions such as ``send``, ``connect``, and ``disconnect`` hiding functionality that is specific to a single client library implementation.
+
+Integration layers
+******************
+
+Table :ref:`Integration layers <integration_layers>` lists the |NCS| client libraries that are supported by the cloud wrapper API togheter with the associated integration layers.
+Each integration layer interacts with a specific client library and contains code that is required to properly setup and maintain a connection to the designated cloud service.
+Based on the Kconfig options listed in :ref:`Integration layers <integration_layers>`, the corresponding integation layer + client library combination is linked in.
```

Comment:
```suggestion
Based on the Kconfig options listed in the table, the corresponding integration layer + client library combination is linked in.
```
Removed the link as you are referring to the table already a couple of lines above.